packages/config/src/main.ts

```diff
@@ -49,6 +49,10 @@ export const resolveConfig = async function (opts) {
     return parsedCachedConfig
   }
 
+  // TODO(kh): remove this mapping and get the extensionApiHost from the opts
+  const extensionApiBaseUrl =
+    host === 'api.netlify.com' ? 'https://api.netlifysdk.com' : `https://api-staging.netlifysdk.com `
```

Comment:
I know we're going to change this mapping at some point. But I would sleep a lot better knowing the fallback was prod not staging, I'd rather staging get broken then prod (I don't think that's the actual staging url, but whatever it is add it there.
```suggestion
    host === 'api.netlify-staging.com' ? https://api-staging.netlifysdk.com ` : 'https://api.netlifysdk.com'
```